buildSrc/LaunchHtml.js

```diff
@@ -1,10 +1,23 @@
 global.window = undefined
 
 function getCspUrls(env) {
+	// we want to have the following allowed connect-src for a given staticUrl like https://app(.local/.test).tuta.com:
+	//
+	// wss://app(.local/.test).tuta.com for websocket
+	// http(s)://*.api(.local/.test).tuta.com for the web app
+	// api://app(.local/.test).tuta.com for the mobile apps api protocol (intercepted by native part)
+	// https://app(.local/.test).tuta.com for the staticUrl itself
+	// https://(local./test.)tuta.com for the website
 	if (env.staticUrl) {
 		const staticUrlParts = env.staticUrl.split("//")
 		const apiUrl = staticUrlParts[0] + "//*.api." + staticUrlParts[1]
-		return `${env.staticUrl} ws${env.staticUrl.substring(4)} ${apiUrl} ${env.staticUrl.replace(/^https?/, "api")}`
+		const webSocketUrl = `ws${env.staticUrl.substring(4)}`
+		const appApiUrl = `${env.staticUrl.replace(/^https?/, "api")}`
+		const websiteUrl = env.staticUrl.includes("app.")
```

Comment:
we have website base url in domain config, why should we do this manipulation at all?